bin/run_docker_dev.sh

```diff
@@ -1,16 +1,18 @@
 #!/bin/bash
 
-# Helper script for running docker in a dev environment. 
+# Helper script for running docker in a dev environment.
 # Use run_docker.sh instead in prod
 
 if [[ $1 == "build" ]]; then
+    mkdir -p devenv/
+    cp test/testdata/* devenv/
```

Comment:
I'd only make the copy if `devenv` is empty. Once you've customized your config, you won't want to always overwrite it with test data.